src/tools/genpolicy/rules.rego

```diff
@@ -339,6 +353,7 @@ allow_linux(p_oci, i_oci) {
 
     allow_masked_paths(p_oci, i_oci)
     allow_readonly_paths(p_oci, i_oci)
+    allow_linux_devices(p_oci.Linux.Devices, i_oci.Linux.Devices)
```

Comment:
@burgerdev , without adding tests for this field, I would be more comfortable initially with just:

is_null(p_oci.Linux.Devices)

But, if you think we should add such tests later, please log an issue as a reminder for that future work.

Reply:
When mounting a block device, both `.devices` and `.OCI.Linux.Devices` are set by the runtime (example request from AKS-CoCo: https://gist.github.com/burgerdev/3a1ae37870e8ee53d603bb9f5d3a2069). I don't know why that's the case, but the test won't work if we don't allow the Linux device here.